runner/options.go

```diff
@@ -510,7 +511,15 @@ func (options *Options) ValidateOptions() error {
 	if len(options.OutputMatchCdn) > 0 || len(options.OutputFilterCdn) > 0 {
 		options.OutputCDN = true
 	}
-
+	// Validate the custom port range
+	if len(options.CustomPorts) > 0 {
+		for _, value := range options.CustomPorts {
+			potentialPorts := sliceutil.Dedupe(strings.Split(value, ","))
+			for _, port := range potentialPorts {
+				customport.ValidateCustomPorts(port)
```

Comment:
here we should check if `customport.ValidateCustomPorts(xx)` returned an error and either propagate/print